lesson3_videos.py

```diff
@@ -50,13 +50,33 @@ class Job(BaseModel):
         Person, related_name='was_filled_by', null = False)
 
 
+class Department(BaseModel):
+    """
+        This class defines Department, which maintains details of in
+        which Department a Person held a Job
+    """
+
+    dept_number = pw.CharField(primary_key=True, max_length=30)
+    dept_name = pw.CharField(max_length=30)
+    dept_manager = pw.CharField(max_length=30)
+    job = pw.ForeignKeyField(
+        Job, related_name='job_name', null=False)
+    query_job_details = Job.select().where(Job.job_name == job)
```

Comment:
Ok, first to change here is that you can't run a query from the definition of a table, the idea of a foreign key is that you can access "related" information through the link created by the foreign key, so lines 64 and 65 are not really required.